tchannel/thrift/generated.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import inspect
+import types
+
+from .util import get_service_methods
+
+
+def from_thrift_module(service, thrift_module):
```

Comment:
Sure. For now we might be able to just keep going down the path I'm doin' now like so:

```
from tchannel import TChannel, from_thrift_module, from_thrift_idl
```